en/dev_log/log_encryption.md

```diff
@@ -0,0 +1,179 @@
+# Log Encryption
```

Comment:
Can we cross link to this _from_ https://docs.px4.io/main/en/dev_log/logging.html ? It's a cool feature so good to make it easier to find.

I was thinking perhaps in the introduction 

> The output log format is [ULog](https://docs.px4.io/main/en/dev_log/ulog_file_format.html)."

 TO 

> The output log format is [ULog](https://docs.px4.io/main/en/dev_log/ulog_file_format.html), and this may also be [encrypted](LINK).

Reply:
I agree. Just pushed that change